packages/eslint-config-globis/package.json

```diff
@@ -20,5 +20,20 @@
   "scripts": {
     "test": "echo test",
     "lint": "echo lint"
+  },
+  "peerDependencies": {
+    "eslint": "^6.7.2",
+    "prettier": "^1.19.1"
```

Comment:
These do not fix the version ?

Reply:
It is not fixed.
because it is free for the user.

Reply:
OK that the user can select version.
However, is disable PeerDependencies setting necessary for renovate app ?

Reply:
aha. done~.
https://github.com/globis-org/frontend-standard/pull/3/commits/4ef67734a6a2da77e19d57605c16a0dcd8375ece